flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -531,6 +541,38 @@
                     .withDescription(
                             "The number of the last buffer size values that will be taken for the correct calculation of the new one.");
 
+    /** The total time for which automated adjusted buffers should be fully consumed. */
+    @Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
+    public static final ConfigOption<Integer> AUTOMATIC_BUFFER_ADJUSTMENT_CONSUMPTION_TIME =
+            ConfigOptions.key(
+                            "taskmanager.network.memory.automatic-buffer-adjustment.consumption-time")
```

Comment:
Somehow `consumption-time` doesn't sound right to me. Maybe something as simple as:
```
taskmanager.network.memory.buffer-debloat.target
```
?